src/Commands/DeployCommand.php

```diff
@@ -29,9 +30,17 @@ class DeployCommand extends Command
      */
     public function handle()
     {
-        Bugsnag::deploy($this->option('repository'), $this->option('branch'), $this->option('revision'));
+        $builderName = $this->option('builder');
+        if (is_null($builderName)) {
+            $process = new Process('whoami');
```

Comment:
Bugsnag does not depend on the process component directly. Also, does this work on Windows?

Reply:
Yeah, whoami works across pretty much everything,